doc/article/technical_article.tex

```diff
@@ -111,19 +111,18 @@
 % ------------------------------------------------------------------------------
 \begin{abstract}
   The main goal of the present article is to describe the idea, goals and main
-  functionalities of the web-based application trckr. We developed trckr for
+  functionalities of the web-based application trckr. The audience for trckr 
   % Since we want trckr to be all lowercase, we avoid using it at the start of
   % the sentence
-  everyone who works on a project and needs an intuitive and simple web tool to
-  accurately track their time spent on d
-ifferent tasks.
+  are people who work on a project and need an intuitive and simple web tool to
+  accurately track their time spent on different tasks.
   % Hint: Empty line makes a paragraph which is semantically different from the
   % forced \\ newline (and much neater)
 
   The back-end is written in Python with the help of the web application
   framework Django and the front-end with the Javascript UI framework Vue.js.
   Both technologies were new to most team members, but all members have proven
-  themselves motivated to learn these skills which was ultimately a benefit to
+  themselves motivated to learn these skills which were ultimately a benefit to
```

Comment:
'was' is with regard to the learning of these skills.  don't know if this change would be right.